tests/init.sql

```diff
@@ -24,4 +24,18 @@ WITH (
 TABLESPACE pg_default;
 
 ALTER TABLE public.qalert_requests
+    OWNER to docker;
+
+CREATE TABLE public.qalert_audits
+(
+    id SERIAL NOT NULL,
+    create_date text 
```

Comment:
prob add not null contraint to create date as well

Reply:
So for some reason, when I put the NOT NULL next to the text, the code didn't work...? I was so confused lmao